<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="50607" started-at="2018-04-12T12:11:35Z" finished-at="2018-04-12T12:12:25Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="50607" started-at="2018-04-12T12:11:35Z" finished-at="2018-04-12T12:12:25Z">
      <class name="testcases.TC01_Login">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:testcases.TC01_Login@178f375]" name="beforeSuite" is-config="true" duration-ms="255" started-at="2018-04-12T17:41:35Z" finished-at="2018-04-12T17:41:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:testcases.TC01_Login@178f375]" name="beforeTest" is-config="true" duration-ms="1" started-at="2018-04-12T17:41:35Z" finished-at="2018-04-12T17:41:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="setValues()[pri:0, instance:testcases.TC01_Login@178f375]" name="setValues" is-config="true" duration-ms="0" started-at="2018-04-12T17:41:35Z" finished-at="2018-04-12T17:41:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setValues -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testcases.TC01_Login@178f375]" name="beforeMethod" is-config="true" duration-ms="6" started-at="2018-04-12T17:41:37Z" finished-at="2018-04-12T17:41:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="login(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testcases.TC01_Login@178f375]" name="login" duration-ms="48262" started-at="2018-04-12T17:41:37Z" data-provider="fetchData" finished-at="2018-04-12T17:42:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Anvar]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[7.1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[sakthivel@testleaf.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Leaf@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[FAILED]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: FAILED
	at utils.Reporter.reportStep(Reporter.java:29)
	at wrapper.WrapperMethods.enterTextByXpath(WrapperMethods.java:304)
	at pages.LoginPage.enterUserName(LoginPage.java:23)
	at testcases.TC01_Login.login(TC01_Login.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testcases.TC01_Login@178f375]" name="afterMethod" is-config="true" duration-ms="3" started-at="2018-04-12T17:42:25Z" finished-at="2018-04-12T17:42:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:testcases.TC01_Login@178f375]" name="afterTest" is-config="true" duration-ms="1" started-at="2018-04-12T17:42:25Z" finished-at="2018-04-12T17:42:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:testcases.TC01_Login@178f375]" name="afterSuite" is-config="true" duration-ms="428" started-at="2018-04-12T17:42:25Z" finished-at="2018-04-12T17:42:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- testcases.TC01_Login -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
